src/index.tsx

```diff
@@ -15,14 +15,14 @@ import "./styles/index.css"
 import "./themes/variables.css"
 
 // Theme overrides for each themes
+import "./themes/acme.css"
 import "./themes/stitch.css"
 import "./themes/luminara.css"
 import "./themes/pug.css"
 
 // CSS style overrides used for workarounds.
 import "./styles/smartscalar-workaround.css"
 import "./styles/dashboard-workaround.css"
-import "./styles/viz-workaround.css"
```

Comment:
This workaround was breaking some dashcards on mobile, and it's no longer needed.

<img src="https://github.com/user-attachments/assets/97604a8d-3ea4-4ca3-885d-4ab8d2d237bd" width="280">